etc/kayobe/kolla/config/octavia.conf

```diff
@@ -0,0 +1,21 @@
+[certificates]
+cert_generator = local_cert_generator
+ca_certificate = /etc/octavia/certs/server_ca.cert.pem
+ca_private_key = /etc/octavia/certs/server_ca.key.pem
+ca_private_key_passphrase = <server CA key passphrase>
+
+[health_manager]
+controller_ip_port_list = 10.208.1.0:5555, 10.208.1.1:5555, 10.208.1.2:5555
+
+[controller_worker]
+client_ca = /etc/octavia/certs/client_ca.cert.pem
+amp_boot_network_list = ef9820af-ea57-4280-a43a-23c4ebd77bc6
+amp_image_tag = amphora
+amp_image_owner_id = 481837de9b39406d93c7782d0d29e48e
+amp_secgroup_list = a11521a8-ea17-4e2e-b8f8-45f152e2979c
+amp_flavor_id = 298272df-846a-4631-954a-e32823402286
+amp_ssh_key_name = octavia
+
+[haproxy_amphora]
+client_cert = /etc/octavia/certs/client.cert-and-key.pem
```

Comment:
where is the client.cert-and-key.pem?

Reply:
Ah, that's left over from when I was looking at the dual CA model, need to take that out of the over-rides. The single CA key is Ansible-Vaulted.